src/Sulu/Bundle/AdminBundle/Metadata/MetadataProviderInterface.php

```diff
@@ -13,5 +13,5 @@
 
 interface MetadataProviderInterface
 {
-    public function getMetadata(string $key, string $locale);
+    public function getMetadata(string $key, string $locale, array $metadataOptions);
```

Comment:
Now that the return value of this function must at least implement the `isCacheable` method, shouldn't we introduce a `MetadataInterface`, which is being returned here? Then accessing this method in the `metadataAction` would also be typesafe.